app/controllers/ops_controller/settings/common.rb

```diff
@@ -218,7 +219,19 @@ def pglogical_save_subscriptions
         add_flash(_("Error during replication configuration save: %{message}") %
                     {:message => bang.message}, :error)
       else
-        add_flash(_("Replication configuration save was successful"))
+        if replication_type == :remote && !params[:exclusion_list].empty?
+          begin
+            new_exclusion_list = params[:exclusion_list].split("\n- ").reject { |s| s.eql?('---') }
```

Comment:
I don't think we should roll our own YAML parsing. I would rather use `YAML.load` and deal with the brakeman issues in some other way.